packages/gui/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@blitzjs/gui",
+  "version": "0.20.0",
+  "scripts": {
+    "start": "blitz start",
+    "build": "blitz build"
```

Comment:
This will run `blitz build` when we build all the other packages for publishing to npm. I don't think this is what we want?

Oh and also `@blitzjs/gui` will be published to npm too. Is that what you are intending?

Reply:
Will it still be published with the `private` field?

Reply:
Oh then no it won't be. But the build script will still be run.

Reply:
Okay, removed the script.